addons/mail/models/mail_thread.py

```diff
@@ -2434,18 +2449,23 @@ def _notify_compute_recipients(self, message, msg_vals):
             # here      : (searching all partners linked to channels with notif email if email is not the author one)
             # TDE FIXME: use email_sanitized
             email_from = msg_vals.get('email_from') or message.email_from
+            email_from = self.env['res.partner']._parse_partner_name(email_from)[1]
```

Comment:
because email_from is using computed field email_formatted of partner 
and it's not comparable (e.g. `'"Mitchell Admin" <admin@example.com>'`)
I think FIXME comment over there is related to this I guess